subprojects/docs/src/docs/release/notes.md

```diff
@@ -112,6 +112,10 @@ The kind folks at [Algolia](https://www.algolia.com/) kindly host an index used
 You can now specify a timeout duration for a task, after which it will be interrupted.
 Read more [about task timeouts](userguide/more_about_tasks.html#sec:task_timeouts) in the docs.
 
+## HTTP retries
+
+When an HTTP call fails, it will be retried a set number of times with exponential backoff. No extra configuration is needed and the backoff time is not configurable at the task level.
```

Comment:
"not configurable at the task level" sounds kinda weird here, I think we should just say "not configurable".

This subsection of the release notes should link to user manual documentation.